core/src/main/scala/kafka/tools/PerfConfig.scala

```diff
@@ -40,20 +40,5 @@ class PerfConfig(args: Array[String]) {
   val showDetailedStatsOpt = parser.accepts("show-detailed-stats", "If set, stats are reported for each reporting " +
```

Comment:
I mean that this config is currently inherited by `ProducerPerfConfig` and it should not be.